### ISSUE TITLE
global is not defined

### ISSUE THREAD

**Issue by rochapablo** [ably/ably-js#603]
It was all working, until I upgrade my node modules.

Follow the error:

```
Uncaught ReferenceError: global is not defined
    at Object../node_modules/ably/browser/static/ably-commonjs.js (vendor.js:143813)
    at __webpack_require__ (runtime.js:80)
    at Module../src/app/components/toolbar/toolbar.component.ts (main.js:5906)
    at __webpack_require__ (runtime.js:80)
    at Module../src/app/components/index.ts (main.js:4870)
    at __webpack_require__ (runtime.js:80)
    at Module../src/app/app.declarations.ts (main.js:663)
    at __webpack_require__ (runtime.js:80)
    at Module../src/app/app.module.ts (main.js:803)
    at __webpack_require__ (runtime.js:80)
```

**My Code**

```
import * as Ably from 'ably';

const ably = new Ably.Realtime(environment.ABLY_PRIVATE_KEY);
const channel = ably.channels.get('mychannel');
channel.subscribe('upload', (message) => {
    //
});
```

Am I missing something?

```
"@angular/core": "^8.1.2",
"ably": "^1.1.15",
```

**Comment by rochapablo** [comment]
Adding ```(window as any).global = window;``` inside polyfills.ts did work.

**Comment by SimonWoolf** [comment]
Hi @rochapablo - thanks for reporting this. Was a regression in the commonjs distribution (which unfortunately isn't covered by automated tests in the same way the normal browser distribution and the nodejs entrypoint are, for some reason -- we'll try to fix that).

Fixed in 1.1.16; if you upgrade to that you can remove that workaround.
